src/test/java/Pages/AllSongsPage.java

```diff
@@ -0,0 +1,126 @@
+//            .visibilityOfElementLocated(By.cssSelector(".all-songs tr.song-item:nth-child(1)")));
+//    actions.contextClick(firstSongInTheList).perform();
+//}
+//public void choosePlayOption(){
+//    wait.until(ExpectedConditions
+//                    .visibilityOfElementLocated(By.xpath("//nav[@class='menu song-menu']//ul//li[@class='playback']//span[1]")))
+//            .click();
+//
+//    //        li.playback
+//    //        //nav[@class='menu song-menu']//ul//li[@class="playback"]
+//    //        //nav[@class='menu song-menu']//ul//li[@class='playback']//span[1]
+//}
+//public boolean isSongPlaying(){
+//    WebElement soundBarVisualizer = wait.until(ExpectedConditions
+//            .visibilityOfElementLocated(By.cssSelector("[data-testid='sound-bar-play']")));
+//    return soundBarVisualizer.isDisplayed();
+//}
+//
+//public WebElement hoverPlay(){
+//    WebElement playBtn = driver.findElement(By.cssSelector("[data-testid='play-btn']"));
+//    actions.moveToElement(playBtn).perform();
+//    return wait.until(ExpectedConditions.visibilityOf(playBtn));
+//}
+//
+//public void choosePlaylistByName(String playlistName){
+//    wait.until(ExpectedConditions
+//                    .visibilityOfElementLocated(By.xpath("//a[contains(text(),'"+playlistName+"')]")))
+//            .click();
+//    ////a[contains(text(),'Playlist to count songs')]
+//}
+//public int countSongs(){
+//    return driver.findElements(By.cssSelector("section#playlistWrapper td.title")).size();
+//}
+//public String getPlaylistDetails(){
+//    return driver.findElement(By.cssSelector("span.meta.text-secondary span.meta")).getText();
+//}
+//public void displayAllSongs() throws InterruptedException{
+//    Thread.sleep(2000);   //Test would not pass without this sleep
+//    List<WebElement> songList = driver.findElements(By.cssSelector("section#playlistWrapper td.title"));
+//    System.out.println("Number of Songs found: " +countSongs());
+//    for (WebElement e : songList){
+//        System.out.println(e.getText());
+//    }
+//}
+//
+//public void doubleClickPlaylist(){
+//    WebElement playlistElement =
+//            wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".playlist:nth-child(3)")));
+//    actions.doubleClick(playlistElement).perform();
+//}
+//public void enterNewPlaylistName(){
+//    WebElement playlistInputField =
+//            wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[name='name']")));
+//    //clear does not work since element has an attribute of required.
+//    playlistInputField.sendKeys(Keys.chord(Keys.CONTROL, "A", Keys.BACK_SPACE));
+//    playlistInputField.sendKeys(newPlaylistName);
+//    playlistInputField.sendKeys(Keys.ENTER);
+//}
+//public String getRenamePlaylistSuccessMsg(){
+//    WebElement notification =
+//            wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("div.success.show")));
+//    return notification.getText();
+//}
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.